### ISSUE TITLE
Chrome 122

### ISSUE THREAD

**Issue by ericmillersfbay** [Starlk7/Magiceden-Launchpad-Bot#6]
Seems it doesn't work with chrome 122 anyway I can fix this?

**Comment by ericmillersfbay** [comment]
Willing to make a small donation if you can get this working before Thursday :)

**Comment by Starlk7** [comment]
Should be fixed. Can you check if it works?

**Comment by Starlk7** [comment]
Try to "npm install chromedriver --chromedriver_version=LATEST" or reinstall

**Comment by ericmillersfbay** [comment]
Yes I'll check it, is that address still also your current one for donations? :)

**Comment by Starlk7** [comment]
Yes <3

**Comment by ericmillersfbay** [comment]
Got this: 

01:52:56| [Thread#1/1 | Wallet#1/1] Error in solanaConnectToPage TimeoutError: Waiting for element to be located By(xpath, //*[contains(@id, "headlessui-dialog-panel")]/div[2]/div/div[2]/div[2]/button/div/div/span[1])
Wait timed out after 5003ms
01:52:56| [Thread#1/1 | Wallet#1/1] Wallet is not connected

guessing because mint is not live yet not sure?

**Comment by Starlk7** [comment]
This should also be fixed by now, try to change solanaController.js in /controllers or just re-download files

**Comment by ericmillersfbay** [comment]
Thank you I think it works, it does ask me to sign my wallet and if i click manually it doesnt work, but it goes away after a sec is that normal?

**Comment by Starlk7** [comment]
Once launched, you no longer need to press anything. When a window with “sign in” pops up, you don’t need to do anything, its will be works

**Comment by ericmillersfbay** [comment]
Also if I set it to Headless: true

[63856:58428:0305/022137.041:ERROR:device_event_log_impl.cc(192)] [02:21:37.041] USB: usb_service_win.cc:105 SetupDiGetDeviceProperty({{A45C254E-DF1C-4EFD-8020-67D146A850E0}, 6}) failed: Element not found. (0x490)

dont know if headless: true is faster or not?

**Comment by Starlk7** [comment]
A little faster, but not very stable, so better to use false
